core/src/main/scala/kafka/common/Topic.scala

```diff
@@ -18,9 +18,14 @@
 package kafka.common
 
 import util.matching.Regex
-import org.apache.kafka.common.internals.TopicConstants.INTERNAL_TOPICS
+
+import scala.collection.immutable
 
 object Topic {
+
+  val GroupMetadataTopicName = "__consumer_offsets"
```

Comment:
Ah, no, that's fine. I just was thinking of the stuff in `TopicConstants` being moved from `main` to `test`, but this is a fair assessment -- the name in the client tests doesn't actually matter.